cli/azd/test/azdcli/cli.go

```diff
@@ -39,6 +39,7 @@ func NewCLI(t *testing.T) *CLI {
 func (cli *CLI) RunCommandWithStdIn(ctx context.Context, stdin string, args ...string) (string, error) {
 	description := "azd " + strings.Join(args, " ") + " in " + cli.WorkingDirectory
 
+	args = append(args, "--debug")
```

Comment:
Added for testing? Should remove before merging?

Reply:
Yes, trying to find out why `azd` is hanging and never ending

Reply:
Removed as it was causing some tests to actually fail.